packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
+| placement     | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                      | ✅       |
+| children      | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element            | ✅       |
+| isOpen        | `boolean`                                                                | `undefined` | Controls the tooltip state                                                |          |
+| defaultIsOpen | `boolean`                                                                | `undefined` | If true, the tooltip will be visible initially                            |          |
```

Comment:
Is there a usecase for this 🤔 i.e. keeping it open on mount in uncontrolled state (a UI should ideally have only one tooltip visible at a time I'm thinking - this way there could be multiple tooltips visible)

Reply:
Discussing this internally.

Reply:
@divyanshu013 discussed this with rastogi, it seems that from design side we discourage opening tooltip by default. 
So I think better to remove it. 

But there is another problem, from design side it's also recommended not to open multiple tooltips at once, if we provide users with isOpen & defaultIsOpen they could potentially open multiples. 

I can think of two solutions: 

1. Treat tooltips like stacks and only allow 1 to be opened even when multiple are triggered

```jsx
<Tooltip content="Tooltip content 1" />
<Tooltip content="Tooltip content 1" />
<Tooltip content="Tooltip content 1" /> // only the last one opens
```
Downside of this approach is that, it's quite unexpected & hard orchestrate for users if they have multiple tooltips in multiple places and files scattered.

 2. Don't allow isOpen & defaultIsOpen. only add events (onVisible, onHidden) for analytics requirements
 
 This is cleaner quoting @saurav12 
 
 > Tooltip is something that should be triggered on demand. Therefore, we should not show multiple tooltips simultaneously.

I think this is a viable approach.

Reply:
Let's re-word to go with approach (2), we can reassess if there are product usecases

Reply:
Will be removing these props so no need of re-word. If someone raises to add isOpen and other props we will justify our thoughts.